### ISSUE TITLE
connection

### ISSUE THREAD

**Issue by cescoferraro** [achiku/dgw#11]
```
cescoferraro@g7: ~/go/src/github.com/b2breservas/api on master [!?]
$ dgw pgsql://postgres:password@localhost/dvdrental?sslmode=disable
2019/05/13 19:15:45 faield to load table definitions: failed to load table def: pq: SSL is not enabled on the server

```

**Comment by kyanny** [comment]
@cescoferraro 
TL;DR `$ dgw postgresql://postgres:password@localhost/dvdrental?sslmode=disable` should work.

I encountered the same issue, but I figured out it. Here is the results of my investigation.

You need to use `postgresql://` or `postgres://` instead of `pgsql://`.

`postgresql://` and `postgres://` are supported by PostgreSQL itself.
You can find it on `33.1.1.2. Connection URIs` section of the official document.
https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING

`psql://` seems provided by `github.com/xo/dburl` library.
You can find it on `Protocol Schemes and Aliases` of its document.
https://github.com/xo/dburl#protocol-schemes-and-aliases

Since `dgw` doesn't use `dburl` library, it doesn't support `pgsql://` URL scheme.

**Comment by achiku** [comment]
Thanks!
